Makefile

```diff
@@ -94,7 +94,9 @@ vet:
 	go vet ./...
 
 .PHONY: build
-build: | $(BINDIR) ## build trust-manager
+build: $(BINDIR)/trust-manager | $(BINDIR) ## build trust-manager for the host system architecture; to cross compile for linux, use "make build-linux-$arch"
+
+$(BINDIR)/trust-manager: | $(BINDIR)
```

Comment:
I think we have to add the go files as a dependency, otherwise make will think that the binary is up-to-date.